api/swagger.yaml

```diff
@@ -2260,7 +2260,9 @@ definitions:
                     ConfigName is the name of the config that this references, but this is just provided for
                     lookup/display purposes. The config in the reference will be identified by its ID.
                   type: "string"
-
+          Isolation:
+            description: "Isolation mode of the containers running the service (default, process, hyperv..."
+            type: "string"
```

Comment:
Perhaps add an enum - what is the behaviour if `""` is passed? Is that the same as `default`?

```
            enum:
              - "default"
              - "process"
              - "hyperv"
```